### PR TITLE
Add ability to report the final status of the job to a pushgateway endpoint

### PR DESCRIPTION
**Pull request by maulikjs**

### CODE REVIEW

File: es_template_manager/es_template_manager.py [hunk 1]
```diff
@@ -67,6 +67,8 @@ def parse_args():
     parser.add_argument('--overwrite-templates', default=False,
                         action='store_true',
                         help='Overwrite any templates that already exist')
+    parser.add_argument('--pushgateway-endpoint', default=N,
```
Comment (gmfrasca) [inline es_template_manager/es_template_manager.py]: default=N? None?  If so, non-required arguments already default to None

File: es_template_manager/es_template_manager.py [hunk 2]
```diff
@@ -84,15 +86,15 @@ def setup_logging(debug):
     return logger
 
 
-def load_template(logger, template_path):
+def load_template(logger, template_path, pushgateway_enpoint):
```
Comment (gmfrasca) [inline es_template_manager/es_template_manager.py]: why include `pushgateway_enpoint` if you aren't going to use it?

**Review by gmfrasca — COMMENTED** [review]
`pushgateway_enpoint` is consistently misspelt throughout the change without the d in endpoint, needs a refactor

**Comment by maulikjs** [comment]
> `pushgateway_enpoint` is consistently misspelt throughout the change without the d in endpoint, needs a refactor

Auto-complete 🤦

**Comment by gmfrasca** [comment]
one small typo but otherwise LGTM

**Pull request merged**